mpa/seg/train.py

```diff
@@ -139,3 +142,69 @@ def train_segmentor(model,
     elif cfg.load_from:
         runner.load_checkpoint(cfg.load_from)
     runner.run(data_loaders, cfg.workflow)
+
+
+# Temporary copy from mmdet
+# TODO: refactor
+def scatter_cpu(inputs):
```

Comment:
This appears to be the same as the one at train.py in classification. If it's the same code, it seems appropriate to transfer it to the utils folder.